src/constants.php

```diff
@@ -30,6 +30,7 @@ function double_map($array)
         'testing' => 0,
         'community' => 1,
         'official' => 2,
+        'featured' => 2,
```

Comment:
This was my original idea, but the map will always return `official` first as it's first in the map. So I don't think we'll ever see `featured` appear on the frontend with this.

Maybe putting `frontend` above `official` will avoid this.

Reply:
Yeah, if featured is meant to replace official, that would make sense. Perhaps I was thinking of something else at the time, but I assumed it would just find the key then pull the value. Also probably would have been a little easier had the system not mixed strings and int but too late now, haha.

Reply:
Also, on my local test it does show Featured instead of Official when searching.